### ISSUE TITLE
loss curve 

### ISSUE THREAD

**Issue by khadijarao78** [spacewalk01/face-mask-detection#2]
hello sir,
Sir I am working on my FYP(Final Year Project) , for this i have used your github file, thank u for this .
sir can u please tell me how can I get the loss curve as you mentioned it on your results . kindly help me.
Thanking you in advance

**Comment by spacewalk01** [comment]
I used wandb dashboard for visualizing results. Wandb is integrated to yolov5. So, you can use it to visualize the loss curve and other training processes, when you are training your yolov5 model. The steps are follows:

1. First, you need to have an account on wandb (I suppose you don't have an account):
Go to [wandb](https://wandb.ai/login) and create an account and login(web server).
3. Then, install wandb using pip in your terminal (enter the following command):
`pip install wandb`
4. Finally, login wandb via your terminal (local-client):
`wandb login`
When you enter this command, it will ask you to enter an API key. Go to [wandb-authorization](https://app.wandb.ai/authorize) and copy the API key and paste it to your terminal. That is it. 

Now, when you train your yolov5 model, it will automatically visualize detection results, losses and accuracies in real time in a [dashboard](https://wandb.ai/home)

**Comment by khadijarao78** [comment]
Thank you for your help sir,
can you please help me for one more thing, sir is this possible that I get  accuracy and loss curve for test results using your code .
kindly help me sir

**Comment by spacewalk01** [comment]
You mean loss and accuracy curves for validation dataset?

**Comment by khadijarao78** [comment]
sir one more i got my loss curve results in this form
![image](https://user-images.githubusercontent.com/106884702/180596472-108fff81-b6d2-4f4c-839f-f56af29b35d5.png)
and in your results file loss curve compare the train and val loss how can I achieve this 

kindly help me

**Comment by spacewalk01** [comment]
You can custom the panel

**Comment by spacewalk01** [comment]
Click on the right top of the panel (three dots) and customize your chart (include a new curve or delete the existing one)
<img width="425" alt="image" src="https://user-images.githubusercontent.com/6389915/180596608-079d9227-4b5f-49ae-a149-0727f12d9948.png">

**Comment by khadijarao78** [comment]
yes loss and accuracy curve of train and validation dataset both

**Comment by khadijarao78** [comment]
![Uploading image.png…]()
sir can you please tell me what does it mean which i mark
